chia/server/server.py

```diff
@@ -606,6 +606,14 @@ def get_outgoing_connections(self) -> List[WSChiaConnection]:
 
         return result
 
+    def get_full_node_outgoing_connections(self) -> List[WSChiaConnection]:
+        result = []
+        connections = self.get_full_node_connections()
+        for connection in connections:
+            if connection.is_outbound:
+                result.append(connection)
+        return result
```

Comment:
For the filtering, there's also
```python
    connections = self.get_full_node_connections()
    outgoing = filter(lambda c: c.is_outbound, connections)
    return list(outgoing)
```

Reply:
@Nikolaj-K I think this makes sense. I think you can write it even more concisely with list comprehension. Maybe you could submit a PR for this?